### DISCUSSION TITLE
Just4Giving Discussions!

### DISCUSSION THREAD

**Discussion by sammou00** [hyf-Group2-fp/Just4Giving#1] [General]
### Landing page : :grin: 
- maybe include a short video explain the project idea  OR more than one short video 
- some text when click on the videos or the slides ......
- sign in and sing up 
- search bar to show the latest items to donate without (giver) information  if possible if not no need 

### About page :
- we could have an about page

**Comment by kemmy72** [comment]
Concerning the repo name: Just4Giving, is also in line with my thought;  another name for consideration :- Givy-Needy (suggestion).
For the logo, suggest something like this:     https://images.app.goo.gl/Um1FoN9fa5d8jLuK9 

Another suggestion for the Backlog under Must-Haves:
To include in Overview of application and Make an account :- 

• The application should allow the user (giver) to delete their account
• The application should allow the user (giver and needer)  with a forget password option
• The application should have a user (giver and needer) comment section

**Comment by bermarte** [comment]
From Anisya:
but the 'just' from the name a bit make me 'hmmm...' :face_with_hand_over_mouth: maybe today we could make vote to really make a decision for the name.
So far the idea that already came up is Resend, things, care, take, gift, take&Give, away, meegive, carebox, nomad, together, just4giving, SOLEX, givly, 2giv, Givfree, just:heart:giving, and I will add 'chinta' (hindi for careness).

There's also a Gratis but is taken (go Gratis).
Let's hear the others.

**Comment by bermarte** [comment]
Final names: meegive givfree just4giving just2giving Givy-Needy just:heart:giving Gratis

**Comment by Brainketunze** [comment]
![image](https://user-images.githubusercontent.com/67590602/116395555-8f377f00-a824-11eb-873e-ccb33b01df51.png)
I found this image for the logo online, I don't kwn whether it will be useful to take in consideration.
from this link:  https://line.17qq.com/articles/euhrcrhx.html

**Comment by AnisyaPurnama** [comment]
## Wednesday stand-up with Dirk and Anthony 28/04/2021

Suggestion:
--------------

- Do more brainstorming, pop up some crazy ideas and then later make them concrete. 
- Write down every thought, idea of the persona (interview the user).
- Maybe narrowing the target? not too abroad.
- If the target is old people then you should have bigger font , bigger UI
- Think more about the different categories Clothes , (food maybe not a good idea )
- Categories are fixed or the user can add more categories ??  
- The  logo is good, the solidarity feel is appears from the logo but maybe the font could be smaller. 
- make different logo versions , the logo need to be good even for very small size 16 px (normal small logo )
- Think also about geo-location, take the address.
- use Wiki to organize all your project

**Comment by sammou00** [comment]
## Brain Storming 29/04/2021

### city :

- Brussels , the capital , it has more migrants. central of Belgium , more refugees , new comers.
- Antwerp
- Gent
- Leuven
- L.L.N
- Mars

### Persona :

- Migrants
- New comers
- Kids under age 18
-

### Catagories:

### User interview :

- connect with some Organization.

---

- contact with some organizations to ask for a feedback (Peyman)
- KringloopWinkel Okazi
- UNHCR Belgium
- Offline updated contains service for people who don't have internet connection
- The app contains should be updated without internet connection
- contact the organization to get updated information

**Comment by sammou00** [comment]
We need to discuss about the possibility of the giver to be become needer and the needer to become giver !!

**Comment by AnisyaPurnama** [comment]
From Samir : 03/05/21

- I have cratered a new folder img`  you can add all the necessary images !!
- for the README it is very easy , I will do it , I have a template! just with a few changes , it will be  good enough for first README then we can change it at the end of the project.
- I have a meeting with Jan  tomorrow ,  and then we will decide which platform to deploy on and which SQL database to chose!!
- UX-UI / Front-End and Back-End if I can help you with anything please let me know!
best regards,

**Comment by kemmy72** [comment]
@ team mates
### Suggestion:

- TERMS OF USE-   By using the app, you consent to the terms of use, if you do not consent; do not use App site. The services provided through our Just4Giving App require you to open an account (sign up) . You are responsible (liable) for maintaining the confidentiality of the information you hold for your account, including your password and activity that occurs under your account; so keep your information confidential. you may be held liable for mis-use or abuse for any other user of our App due to someone else using your email, password or account as a result of your fault to keep your account information confidential.  Any feedback (comment) you provided on our site shall be regarded as non-confidential, will be free to use such information uninhibitedly.

### Suggestion:

- Privacy Policy --  Take a moment to read through our privacy policy, is all about how information and your personal data is shared between the giver(donor) and needer. There is chance to assess the category listings of available items (goods) before enabling or sign-up or registering. You can view information at any time.

**Comment by sammou00** [comment]
### Daily Standup  4//05/2021
Dear all,

Wednesday : 10:30 AM 
---
- @bermarte  we need to meet tomorrow morning to discuss about the milestones. 
- for the landing page @DIVYASREE345  you can fork the repo create a new branch `1-landingpage` and push the code .
- @AnisyaPurnama   if you need help let us know , better be a member from the `Front-End` because,he/she knows better, if no one can , I will do it.
- ---
Thursday  : 
- I will be busy Thursday morning until 2 PM (I won't be available).

- I have feeling that I am not helping you enough!! if this is the case , please let me know , so I can help you more!!

Best regards,

**Reply by AnisyaPurnama** [reply]
Samir you did a great job for our team! really!

**Comment by sammou00** [comment]
### 09/05/2021 meeting With @Dirk and @Anthony 
---


## Front-End:

1- there is already a lot of information about sign in and sign out in the boilerplate repo

2- Front-End so far good to go 

3- different color for the text 


## UX/UI
4- Fimga design : need to meet with Front and the back Ends 

5- add many image to one item ? or only one picture 

6- quantity ? good or maximize the number of objects ? ++ -- better 

7- edit and delete buttons on hover 

8- `Item` word no need in UX/UI

9- show only the categories then the item 

10- shadow the text

## Back-End:

11 - quantity for the goods for many needers 

12- modify the DB schema
